### PR TITLE
feat: give an indication when profile is updated

### LINKED ISSUE #317: [FEATURE] give an indication when profile is updated
### Description

When a user updates their profile, they get an indication that it's updated, just a text under the delete account button.

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by 9josky**
Fixes Issue

closes #317

# 👨‍💻 Changes proposed(What did you do ?)

-  added a text indication so user can know when their profile has been updated.

# ✔️ Check List (Check all the applicable boxes)

- [] My code follows the code style of this project.
- [] This PR does not contain plagiarized content.
- [] The title and description of the PR is clear and explains the approach.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -56,7 +54,7 @@ function App() {
             clientId={clientID}
             domain="https://whisper.kinde.com"
             logoutUri={window.location.origin}
-            redirectUri={window.location.origin + '/login'}
+            redirectUri={window.location.origin}
```
Comment (Dun-sin) [inline client/src/App.jsx:57]: this has nothing to do with your change, revert back everything in this file

File: client/src/pages/Login.jsx
Comment (Dun-sin) [inline client/src/pages/Login.jsx:1]: revert everychange here

File: client/src/pages/Profile.jsx
Comment (Dun-sin) [inline client/src/pages/Profile.jsx:1]: install prettier in your VS Code and run it in this file

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/pages/Login.jsx
Comment (9josky) [reply client/src/pages/Login.jsx:1]: No changes where made there it was my formatter acting up

File: client/src/App.jsx (hunk 1 above)
Comment (9josky) [reply client/src/App.jsx]: Sorry about that resolved, when i was having issues with login in using kinde that was when i removed the plus login but forgot to undo

File: client/src/pages/Profile.jsx
Comment (9josky) [reply client/src/pages/Profile.jsx:1]: did that

File: client/src/App.jsx
Comment (Dun-sin) [inline client/src/App.jsx:1]: i don't want this file in your changes

File: client/src/pages/Login.jsx
Comment (Dun-sin) [inline client/src/pages/Login.jsx:1]: same with this one

File: client/src/pages/Profile.jsx
Comment (Dun-sin) [inline client/src/pages/Profile.jsx:1]: look you may have to close this PR and i just want to see changes in this file and even that just the part that has to do with your PR, you can google some solutions, maybe update your prettier?

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**